README.md

```diff
@@ -11,6 +11,31 @@ Or add to your `requirements.txt`/`setup.py` file the following:
 git+https://github.com/materialscloud/mz-bokeh-package.git@master
```

Comment:
In the line here above, please also add "pyproject.toml", in addition to requirements.txt and setup.py.